modules/administration_manual/pages/configuration/files/encryption/index.adoc

```diff
@@ -0,0 +1,132 @@
+
+Encryption keys are stored only on the ownCloud server, eliminating exposure of your data to third-party storage providers. 
+The encryption application does *not* protect your data if your ownCloud server is compromised, and it does not prevent ownCloud administrators from reading users' files.
+
+This would require client-side encryption, which this application does not provide. 
+If your ownCloud server is not connected to any external storage services, it is better to use other encryption tools, such as file-level or whole-disk encryption.
+
+[CAUTION]
+====
+SSL terminates at or before the webserver on the ownCloud server.
+Consequently, all files are in an unencrypted state between the SSL connection
+termination and the ownCloud code that encrypts and decrypts them.
+This is, potentially, exploitable by anyone with administrator access to your server.
+For more information, read: https://owncloud.org/blog/how-owncloud-uses-encryption-to-protect-your-data/[How ownCloud uses encryption to protect your data].
+====
+
+include::encryption-types.adoc[]
+
+[[before-enabling-encryption]]
+== Before Enabling Encryption
+
+Plan very carefully before enabling encryption, because it is not
+reversible via the ownCloud Web interface. If you lose your encryption
+keys, your files are *not* recoverable. Always have backups of your
+encryption keys stored in a safe location, and consider enabling all
+recovery options.
+
+You have more options via xref:configuration/server/occ_command.adoc#encryption[the `occ` command's encryption options].
+
+[IMPORTANT]
+====
+You can't manage encryption without access to the command line.
+If your ownCloud installation is on a hosted environment and you don't have access to the command line, you won't be able to run xref:configuration/server/occ_command.adoc[occ commands].
+In this case, *don't enable encryption*!
+====
+
+include::enable-encryption.adoc[leveloffset=+1]
+
+[[files-not-encrypted]]
+== Which Files Are Encrypted
+
+Only the data in the files in `data/<user>/files` are encrypted, and not the filenames or folder structures. 
+The following files are *never* encrypted:
+
+* Existing files in the trash bin & Versions. 
+  Only new and changed files after encryption is enabled are encrypted.
+* Existing files in Version
+* Image thumbnails from the Gallery app
+* Previews from the Files app
+* The search index from the full-text search app
+* Third-party app data
+
+There may be other files that are not encrypted. 
+
+NOTE: Only files that are exposed to third-party storage providers are guaranteed to be encrypted.
+
+== Encrypting External Mountpoints
+
+You and your users can encrypt individual external mount points. 
+You must have external storage enabled on your Admin page, and enabled for your users. 
+Encryption settings can be configured in the mount options for an external storage mount; see link:configuration/files/external_storage_configuration_gui.adoc#mount-options[Mount Options].
```

Comment:
only a small thing, 
pls use xref instead of link as we have done for other internal references.